packages/buidler-core/test/internal/buidler-evm/stack-traces/compilation.ts

```diff
@@ -38,14 +47,49 @@ function getSolcInput(
   };
 }
 
-export function compile(
+function loadCompilerSources(compilerPath: string) {
```

Comment:
I'd add a comment here explaining that this was copied from `compiler/index.ts`, as it's very hacky.